src/AutoMapper/Configuration/CtorParamConfigurationExpression.cs

```diff
@@ -43,5 +43,10 @@ public void Configure(TypeMap typeMap)
                 action(parameter);
             }
         }
+
+        public bool CheckCtorParamName(string paramName)
```

Comment:
This just hides the meaning. Just expose the parameter name.

Reply:
Update made